### ISSUE TITLE
False positive E701 multiple statements on one line (colon) w/ "class" in variable name

### ISSUE THREAD

**Issue by shapiromatron** [PyCQA/pycodestyle#772]
The following incorrectly gives a 701 warning:

```python
classifiers: int = 2
```
Looks like if the variable name starts with `class` and contains a colon it might trigger the error
https://github.com/PyCQA/pycodestyle/blob/master/pycodestyle.py#L1131

**Comment by sigmavirus24** [comment]
@shapiromatron I believe this has been fixed but you've provided no information as to what version you're using so I can't even begin to try to replicate what you're seeing.

**Comment by shapiromatron** [comment]
My apologies, and thanks for the quick response. Sorry, I'm using flake8 and therefore an older version of pycodestyle; it's been fixed in the new version:

```bash
$ pipenv run pip freeze
pycodestyle==2.3.1

$ pipenv run pycodestyle test.py
test.py:5:12: E701 multiple statements on one line (colon)

$ pipenv run pip install -U pycodestyle
Collecting pycodestyle
  Using cached https://files.pythonhosted.org/packages/e5/c6/ce130213489969aa58610042dff1d908c25c731c9575af6935c2dfad03aa/pycodestyle-2.4.0-py2.py3-none-any.whl
Installing collected packages: pycodestyle
  Found existing installation: pycodestyle 2.3.1
    Uninstalling pycodestyle-2.3.1:
      Successfully uninstalled pycodestyle-2.3.1
Successfully installed pycodestyle-2.4.0
 
$ pipenv run pycodestyle test.py
$ 
```

Ok to close; sorry for time lost in trying replicating, and thank you for your work!
